app/models/searchable.rb

```diff
@@ -150,6 +150,6 @@ def search_index
   end
 
   def can_index_in_search?
-    self.class.searchable_instances.find_by(id: self.id).present? && Whitehall.searchable_classes.include?(self.class)
+    self.class.searchable_instances.find_by(id: self.id).present? && RummagerPresenters.searchable_classes.include?(self.class)
```

Comment:
It is, but I didn't want to change too much in this PR. This is about as much as I'm comfortable deploying in one go as there is a lot of complexity in this code.

I would rather get rid of `Searchable` entirely. It doesn't need to be part of the models, it's really difficult to extend, and whitehall models already includes so many mixins that you can't find where anything is defined. There is a massive whitelist of `SEARCH_FIELDS` defined there, so whenever you want to reuse it for something new, you have to go back and change that.

I think the logical next step after this PR would be to replace the `search_index` method that gets mixed into the model classes with presenters. That pattern worked well for the publishing api integration. The Searchable module confuses me even looking at it now because it's so difficult to read, and this is much more of a problem than duplication for me because it wastes so much time whenever you need to change something in this area.